test/statement_test.ts

```diff
@@ -30,9 +30,9 @@ describe('statements', () => {
   });
   it('translates try/catch', () => {
     expectTranslate('try {} catch(e) {} finally {}')
-        .to.equal(' try { } catch ( e ) { } finally { }');
+        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
```

Comment:
To be precise, what I'm proposing long-term is that we use the common syntax in both Dart and JS and access stack via a facade (because the field names are different; it's `err.stack` vs `err.stackTrace`). Example:

```dart
// facade/error.dart
class ErrorWrapper {
  static stackTrace(Error err) => err.stackTrace;
}
```

```typescript
// facade/error.ts
export class ErrorWrapper {
  static stackTrace(err:Error) {
    return err.stack;
  }
}
```

Then in all places in Angular 2 code we would:

```typescript
try {
  ...
} catch(err) {
  var st = ErrorWrapper.stackTrace(err);
}
```

Works in both languages.